### ISSUE TITLE
check_duplicate_binaries should exclude nosrc in addition to src

### ISSUE THREAD

**Issue by lnussel** [openSUSE/osc-plugin-factory#990]
https://build.opensuse.org/package/view_file/openSUSE:Leap:42.3:Staging/dashboard/duplicate_binaries has too much noise, e.g.:

  FlightGear-data:
  - FlightGear-data
  - FlightGear-data

huh? :-)

**Comment by jberry-suse** [comment]
Was introduced when I removed the filter for only binaries matching arch.

```
FlightGear-data-2017.1.2-2.1.noarch.rpm
FlightGear-data-2017.1.2-2.1.nosrc.rpm
```

Looks like you resolved in #993 with `if parch in ('src', 'nosrc'):` instead of just `!= 'src'`.

**Comment by jberry-suse** [comment]
This would appear to have been completed in 2bbbd6f4656ba3f260cb60cb69640389b3035c9d, but never closed.
